File: P9U4.cs
using System;

class Program
{
    static void Main()
    {
        string[] nombres = { "Luna", "Ishai", "Ricardo", "Mayela" };
        string[] apellidos = { "Paez", "Juarez", "Vega", "Cerda" };

        string[] NombresCom = Nombres_Apellidos(nombres, apellidos);

        Console.WriteLine("Nombres completos:");
       Arreglo(NombresCom);
    }

    static string[] Nombres_Apellidos(string[] nombres, string[] apellidos)
    {
        string[] Nombres_Com = new string[nombres.Length];

        for (int i = 0; i < nombres.Length; i++)
        {
            Nombres_Com[i] = $"{nombres[i]} {apellidos[i]}";
        }

        return Nombres_Com;
    }

 
    static void Arreglo(string[] arreglo)
    {
        foreach (var elemento in arreglo)
        {
            Console.WriteLine(elemento);
        }
    }
}

File: P9U4.java
package com.mycompany.P9U4;

public class P9U4 {
    public static void main(String[] args) {
        String[] nombres = { "Luna", "Ishai", "Ricardo", "Mildred" };
        String[] apellidos = { "Paez", "Juarez", "Ruiz", "Vega" };

        String[] nombres_Completos = nombres_Apellidos(nombres, apellidos);

        System.out.println("Nombres completos:");
        imprimir_Arreglo(nombres_Completos);
    }

    static String[] nombres_Apellidos(String[] nombres, String[] apellidos) {
        String[] nombres_Completos = new String[nombres.length];

        for (int i = 0; i < nombres.length; i++) {
            nombres_Completos[i] = nombres[i] + " " + apellidos[i];
        }

        return nombres_Completos;
    }

    static void imprimir_Arreglo(String[] arreglo) {
        for (String elemento : arreglo) {
            System.out.println(elemento);
        }
    }
}